### PR TITLE
Makefile: update targets for presubmit jobs

### PR DESCRIPTION
**Pull request by nikhita**
**What this PR does / why we need it**:

For #2520 and https://github.com/kubermatic/infra/pull/111

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by alvaroaleman** [comment]
@nikhita You must add the ssh clone url on every single presubmit because via HTTPS the user doesn't have any permissions.

**Comment by nikhita** [comment]
Where did the logs go O_o

**Comment by nikhita** [comment]
/retest

Please work...

**Comment by alvaroaleman** [comment]
@nikhita It would make sense to also remove everything that is now in Prow from the  `.drone.yaml`

**Comment by nikhita** [comment]
/test pull-kubermatic-lint

:crossed_fingers:

File: api/hack/verify-swagger.sh [hunk 1]
```diff
@@ -4,15 +4,29 @@ set -o errexit
...
+mkdir -p "${TMP_DIR}/bin"
 
 cd ${API_DIR}/vendor/github.com/go-swagger/go-swagger/cmd/swagger
-go install
+env "GOBIN=${TMP_DIR}/bin" go install
+export PATH="${TMP_DIR}/bin:${PATH}"
+
 cd ${API_DIR}/cmd/kubermatic-api/
-swagger generate spec --scan-models -o ${TMP_SWAGGER}
+swaggger generate spec --scan-models -o ${TMP_SWAGGER}
```
Comment (nikhita) [inline api/hack/verify-swagger.sh]: :woman_facepalming:

**Comment by nikhita** [comment]
The e2e job is still running but all other ones are green. :green_heart:

**Comment by alvaroaleman** [comment]
Finally :))))

/approve
/lgtm

**Comment by nikhita** [comment]
/assign @p0lyn0mial 
for approval

File: api/pkg/controller/rbac/sync_resource_test.go [hunk 2]
```diff
@@ -502,16 +502,6 @@ func TestEnsureDependantsRBACRole(t *testing.T) {
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
 			// setup the test scenario
-			kubermaticObjs := []runtime.Object{}
-			projectIndexer := cache.NewIndexer(cache.MetaNamespaceKeyFunc, cache.Indexers{})
-			if test.existingProject != nil {
-				err := projectIndexer.Add(test.existingProject)
```
Comment (p0lyn0mial) [inline api/pkg/controller/rbac/sync_resource_test.go:508]: does it mean that `test.existingProject` in all test scenarios are not used and can be removed ?

Comment (nikhita) [reply api/pkg/controller/rbac/sync_resource_test.go:508]: Yes, removed. :+1:

**Comment by nikhita** [comment]
Lint test passed locally, but just to double check:

/test pull-kubermatic-lint

**Pull request merged**